src/types/sites.ts

```diff
@@ -8,3 +8,7 @@ export interface SiteData {
 export interface SiteDataRequest {
   siteNames: SiteData[]
 }
+
+export interface SitePreviewRequest {
+  imageUrl: string
```

Comment:
our backend DTO is `imageUrl?` while this is defined as `imageUrl`. Should we make them consistent?

Reply:
okay